lms/djangoapps/discussion/rest_api/api.py

```diff
@@ -1626,7 +1626,7 @@ def get_course_discussion_user_stats(
 
     """
     course_key = CourseKey.from_string(course_key_str)
-    is_privileged = has_discussion_privileges(user=request.user, course_id=course_key)
+    is_privileged = has_discussion_privileges(user=request.user, course_id=course_key) or request.user.is_staff
```

Comment:
What is the use case for the validation error?

Reply:
It is to restrict users without privileges to access stats based on **FLAGS**!